cmd/snap-confine/snap-confine.c

```diff
@@ -386,8 +386,41 @@ int main(int argc, char **argv)
 		    " but should be. Refusing to continue to avoid"
 		    " permission escalation attacks");
 	}
+
+	/* perform global initialization of mount namespace support for confined
+	 * snaps only, or both when parallel-instances feature is enabled */
```

Comment:
This comment isn't perfect with the conditional, since devmode isn't mentioned and 'or both' isn't right. If you are doing another round of commits for this, please adjust this accordingly. Not a blocker.